components/notebook-crd/cmd/manager/main.go

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
we probably want to name this components/notebook-controller since this is what it is doing

Reply:
makes sense, renamed